src/main/java/com/GeorgesServer/app/RequestLogger.java

```diff
@@ -0,0 +1,23 @@
+package com.GeorgesServer.app;
+
+import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
+
+import java.util.ArrayList;
+
+public class RequestLogger {
+
+    private ArrayList<String> logs;
+
+    public RequestLogger() {
+        this.logs = new ArrayList<>();
```

Comment:
So these logs exist in memory which means that as soon as your server is shut down this dat disappears which kind of defeats the purpose of a log. Typically, you'd want to have a file that you output to or something that has a bit more persistence. Just something to keep in mind.